cluster-provision/images/vm-image-builder/build-containerdisk.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+
+SCRIPT_DIR="$(
```

Comment:
Not sure what you are doing here, but sub-shelling is wasteful and looks strange.
Maybe this will help:
Getting the current folder (of where the script exists): `EXEC_PATH=$(dirname "$(realpath "$0")")`
Getting one folder above: `PARENT_PATH="$(dirname $EXEC_PATH)"`
Getting a folder under the current one: `FOO_PATH="$PWD/foo"`

Reply:
I will simplify it in following commits.